cloto/manager/AuthorizationManager.py

```diff
@@ -36,47 +39,96 @@ class AuthorizationManager():
     """
     myClient = None
     client = requests
+    session = None
 
-    def generate_adminToken(self, username, password, url):
-        """This method generates an admin token."""
+    def generate_session(self, username, password, tenant_id, tenant_name, user_domain_name, auth_api, url):
```

Comment:
Shouldn't this be named `get_auth_token`? No matter we use or not sessions, this function is intended to retrieve such token.